client/pages/park-details.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import parseRoute from '../lib/parse-route';
+
+export default class ParkDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      park: '',
+      isLoading: true,
+      path: ''
+    };
+  }
+
+  componentDidMount() {
+    const { parkCode } = this.props;
+    const [ path ] = window.location.hash.split('&');
```

Comment:
you can have App pass this component a `path` prop instead of reaching out to window location. Think of it as App earning its keep.